lib/authorizer/oauth2.js

```diff
@@ -1,3 +1,10 @@
+var HEADER = 'header',
+    URL = 'url',
+    DEFAULT_ADD_TOKEN_TO = HEADER,
```

Comment:
?

Reply:
a token can be added to the header or to the URL
the default value will be `header`, if not specified on [line 64](https://github.com/postmanlabs/postman-runtime/pull/384/files#diff-7385a12d89075d66ee2143cd2e242ba5R64)

Reply:
So use `HEADER` there instead.